### DISCUSSION TITLE
Python Django deployment

### DISCUSSION THREAD

**Discussion by kb-nett** [vercel/community#6499] [Feature Question]
### Question

Hi everyone,
I am new to Vercel but what I have read so far is that with Vercel I can deploy JavaScript frameworks and also Python as edge functions. What I couldn't find is whether I can also deploy a Django based web app on Vercel? If it's possible then please guide me to its documentation and help.
Thanking you in advance.
Kamran.

**Comment by versecafe** [comment]
@kb-nett It's doable and there is even an available template https://vercel.com/templates/python/django-hello-world, which I would recommend pumping the dependencies on to make sure you're running Django 5, just be aware that cold start times can causes first time runs of functions to take up to two seconds, but subsequent runs are faster and this can be side stepped through function warming.

**Comment by kb-nett** [comment]
Yes I have tried using this demo but it is a very basic simple django application that doesn't use formal django templating engine and actually works as serverless function. When I try to deploy my django app, build is successful but runtime shows '404- Not Found' error. Inspecting the deployment logs shows "No framework detected".  Apparently vercel deployment fails to detect Django framework ☹ .
________________________________
